pcapgo/capture_test.go

```diff
@@ -3,20 +3,31 @@
 // Use of this source code is governed by a BSD-style license
 // that can be found in the LICENSE file in the root of the source
 // tree.
-//go:build linux && go1.9
-// +build linux,go1.9
+//go:build linux
 
 package pcapgo_test
 
 import (
+	"bytes"
+	"context"
+	"fmt"
 	"log"
 	"os"
+	"sync/atomic"
+	"testing"
+	"time"
+
+	"github.com/vishvananda/netlink"
```

Comment:
in my opinon, if we need to have the `netlink` dependency, we should make it clear that it must stay contained within testing.

Reply:
sure, although it's not new, it's in the `  go.mod`   since June 21 :sweat_smile: